src/OrchardCore/OrchardCore.Search.AzureAI.Core/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -16,22 +13,10 @@ namespace OrchardCore.Search.AzureAI;
 
 public static class ServiceCollectionExtensions
 {
-    public static bool TryAddAzureAISearchServices(this IServiceCollection services, IShellConfiguration configuration, ILogger logger)
+    public static IServiceCollection AddAzureAISearchServices(this IServiceCollection services)
     {
-        var section = configuration.GetSection("OrchardCore_AzureAISearch");
-
-        var options = section.Get<AzureAISearchDefaultOptions>();
-        var configExists = true;
-        if (string.IsNullOrWhiteSpace(options?.Endpoint) || string.IsNullOrWhiteSpace(options?.Credential?.Key))
-        {
-            configExists = false;
-            logger.LogError("Azure AI Search module is enabled. However, the connection settings are not provided in configuration file.");
```

Comment:
Why is this check not useful anymore?

Reply:
It was useful when I implemented the module. After the module evolved, it is no longer needed since we load the settings in a configuration file and it'll tell us where the configuration come from (file or Settings)